### DISCUSSION TITLE
Move windows without resizing

### DISCUSSION THREAD

**Discussion by iamvery** [rxhanson/Rectangle#415] [Ideas]
First of all, please pardon me if this has already been requested or worse is already a feature that I was unable to figure out 😅 

I would like to have the ability to move windows to corners _without_ them being resized. I can imagine a few different ways to implement this.

1. A new set of "corner" options that move windows sans resizing.
2. An app-specific configuration that disables resizing when moving windows.
3. A global preference that disables resizing when moving windows.

I don't have a strong preference, but I sort of feel like the 1st option would be the most obvious and flexible. Do you think this would be a worthwhile feature? I am willing to work on the implementation if so.

Thank you for the great app!

**Comment by rxhanson** [comment]
The easiest way is to set this setting: https://github.com/rxhanson/Rectangle#move-updownleftright-dont-center-on-edge, then just execute a couple of move shortcuts to get it in the corner you want.

If you want one shortcut for a corner without resizing (the equivalent of your 1st option) this is not something I have plans to add in to Rectangle at this time, although it's possible to build this with the custom shortcut creator in [Hookshot](https://hookshot.app), my paid superset of Rectangle.

If you wanted to create a pull request for a terminal command setting to disable the resizing for just the corner commands, I don't see a reason why that couldn't be added in.

**Comment by iamvery** [comment]
Thank you so much for the reply and the pointer about disabling centering on edge. That's perfect for my use case! I really appreciate you pointing me in the right direction 🙏. I may give a pull request a shot eventually to get reacquainted with XCode. Thank you for confirming that this would be an acceptable change to consider.

Have a great day 🙂

**Comment by johansolve** [comment]
Excellent tip about disabling centering on edge! 

For reference the documentation of it has moved to https://github.com/rxhanson/Rectangle/blob/master/TerminalCommands.md#disabling-window-restore-when-moving-windows
